.github/actions/custom-actions/teguio_staging_custom/action.yml

```diff
@@ -29,6 +29,22 @@ runs:
       with:
         ssm_parameter: "STG_TWILIO_CO_TWITTER_FLEX_FLOW_SID"
         env_variable_name: "TWITTER_FLEX_FLOW_SID"
+    # Set Instagram variables
+    - name: Set Aselo Facebook App Secret
+      uses: "marvinpinto/action-inject-ssm-secrets@latest"
+      with:
+        ssm_parameter: "FACEBOOK_APP_SECRET"
+        env_variable_name: "FACEBOOK_APP_SECRET"
+    - name: Set helpline Facebook Page Access Token
+      uses: "marvinpinto/action-inject-ssm-secrets@latest"
+      with:
+        ssm_parameter: "FACEBOOK_PAGE_ACCESS_TOKEN_102132102625343_Te-Gu-o---Staging"
```

Comment:
For anybody else looking at this PR, this replaces is the result of replacing spaces and weird characters (like `í`) for dashes. That way AWS won't complain :)